backend/src/users/decorators/get-user.decorator.ts

```diff
@@ -1,7 +1,8 @@
 import { createParamDecorator, ExecutionContext } from '@nestjs/common';
```

Comment:
This get-user decorator might be redundant since the jwtguard you created (via the jwt.strategy.ts) already returns the user document I think. In the group and expense resources, when implementing the jwt auth guard, I can isolate and utilize the user document in my services (although I typically just use the user's id).